content/en/docs/notebooks/troubleshoot.md

```diff
@@ -60,32 +60,42 @@ kubectl get pod -o yaml ${NOTEBOOK}-0
 jupyter-${USERNAME}
```

Comment:
Since these are single line  code-blocks, can you just make them inline (not triple quotes), its quite messy right now.

For example, in this case you would do:
```
* If you are using username/password auth with Jupyter the pod will be named: `jupyter-${USERNAME}`
```
Note: I referring to the non-shell ones.

Reply:
Great suggestion. I changed it to inline, can you please take another look?

Reply:
looks good